src/Core/Analysis/INotifyQualityProfilesChanged.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+using System;
+
+namespace SonarLint.VisualStudio.Core.Analysis
+{
+    public interface INotifyQualityProfilesChanged
+    {
+        /// <summary>
+        /// Notification that one or more Quality Profiles have changed
+        /// </summary>
+        event EventHandler QualityProfilesChanged;
```

Comment:
The `IAnalysisConfigMonitor` implementation will listen for this event.

The implementation of `AnalysisConfigMonitor` is in currently in `Integration.VSIX`. However, all of its dependencies are in `Core` so the implementation can be moved into `Core`. I was thinking about doing that first, but I'll do it separately after this PR.